.thoth.yaml

```diff
@@ -0,0 +1,12 @@
+host: khemenu.thoth-station.ninja
+tls_verify: false
+requirements_format: pipenv
+
+runtime_environments:
+  - name: 'ubi:8'
```

Comment:
shouldn't this be changed to `rhel:8.0`?

Reply:
@fridex

Reply:
Oh, yes... I'll adjust PRs. Thanks for spotting this :+1:

Reply:
thanks for raising all the PR's